src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -436,4 +436,18 @@ class StockMovement {
         return stockMovement
     }
 
+    static def buildCsvRow(StockMovementItem lineItem = null) {
```

Comment:
I don't feel confident about the name for this, since it sounds more reasonable for exporting part, and for import to get the column size I just do:
```groovy
StockMovement.buildCsvRow().keySet().size()
```
so let me know if it should stay or you have any other suggestion for the name of this